tracetools_launch/tracetools_launch/action.py

```diff
@@ -114,6 +114,7 @@ def __init__(
         context_fields:
             Union[Iterable[SomeSubstitutionsType], Dict[str, Iterable[SomeSubstitutionsType]]]
             = names.DEFAULT_CONTEXT,
+        subbuffer_size: int = 8 * 4096,
```

Comment:
It should also be added to `parse()` to support XML and YAML launch files. A `@property` getter should be added, too.

Reply:
done